src/ray/raylet/worker_pool.cc

```diff
@@ -238,7 +238,7 @@ Process WorkerPool::StartWorkerProcess(const Language &language,
         break;
       }
       case Language::JAVA: {
-        code_search_path_str = "-Dray.job.code-search-path" + code_search_path_str;
+        code_search_path_str = "-Dray.job.code-search-path=" + code_search_path_str;
```

Comment:
How about add a test case? I am confused that it seems to be an obvious problem, but it hasn't happened before. Thanks.

Reply:
I run a demo in ray cluster and set code-search-path in ray.conf。 but it failed.  I find start worker process command  wrong.